whisper-finetune/inference_endpoint/score.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from onnxruntime import InferenceSession
+import os
+from transformers import WhisperProcessor
+import json 
+
+# Documentation: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints
+# Troubleshooting: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-troubleshoot-online-endpoints
+  
+# The init() method is called once, when the web service starts up.
+def init():  
+    global sess
+    global processor
```

Comment:
global variables can be named with all caps.

Reply:
Done.